api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -193,6 +193,7 @@ public PersonAttributeType savePersonAttributeType(PersonAttributeType type) thr
 	
 	/**
 	 * @see org.openmrs.api.PersonService#retirePersonAttributeType(PersonAttributeType, String)
+
```

Comment:
please check your IDE formatter settings and then let the formatter take care of these issues here.